jquery.flot.js

```diff
@@ -1360,7 +1360,7 @@ Licensed under the MIT license.
             } else if (norm < 3) {
                 size = 2;
                 // special case for 2.5, requires an extra decimal
-                if (norm > 2.25 && (tickDecimals == null || dec + 1 <= tickDecimals)) {
+                if (norm > 2.25 && (tickDecimals == null || (dec + 1) <= tickDecimals)) {
```

Comment:
Same for this function: moving the param checkings at the beginning of the function should make the rest of the body simpler to read.